index.html

```diff
@@ -1,318 +1,325 @@
 <!DOCTYPE html>
-<meta charset='utf-8'>
-<title>Web Serial API</title>
-<script class='remove' src='https://www.w3.org/Tools/respec/respec-w3c' async></script>
-<script class='remove' src="respecConfig.js"></script>
-<link rel="stylesheet" href="styles/spec.css">
-<section id='abstract'>
-The <cite>Serial API</cite> provides a way for websites to read and write from a
-serial device through script. Such an API would bridge the web and the physical
-world, by allowing documents to communicate with devices such as
-microcontrollers, 3D printers, and other serial devices.
-
-There is also a companion <a
-href="https://github.com/WICG/serial/blob/main/EXPLAINER.md">explainer</a>
-document.
-</section>
-
-<section id='sotd'>
-  This is a work in progress. All <a
-  href="https://github.com/WICG/serial">contributions</a> welcome.
-</section>
-
-<section data-dfn-for="Navigator">
-  <h2>Extensions to the {{Navigator}} interface</h2>
-
-  <pre class="idl">
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>
+      Web Serial API
+    </title>
+    <script class='remove' src='https://www.w3.org/Tools/respec/respec-w3c'
+    async></script>
+    <script class='remove' src="respecConfig.js"></script>
+    <link rel="stylesheet" href="styles/spec.css">
+  </head>
+  <body>
+    <section id='abstract'>
+      The <cite>Serial API</cite> provides a way for websites to read and write
+      from a serial device through script. Such an API would bridge the web and
+      the physical world, by allowing documents to communicate with devices
+      such as microcontrollers, 3D printers, and other serial devices. There is
+      also a companion <a href=
+      "https://github.com/WICG/serial/blob/main/EXPLAINER.md">explainer</a>
+      document.
+    </section>
+    <section id='sotd'>
+      This is a work in progress. All <a href=
+      "https://github.com/WICG/serial">contributions</a> welcome.
+    </section>
+    <section data-dfn-for="Navigator">
+      <h2>
+        Extensions to the {{Navigator}} interface
+      </h2>
+      <pre class="idl">
     [Exposed=Window, SecureContext]
     partial interface Navigator {
       [SameObject] readonly attribute Serial serial;
     };
   </pre>
-
-  <h3><dfn>serial</dfn> attribute</h3>
-
-  When getting, the {{Navigator/serial}} attribute always returns the same
-  instance of the {{Serial}} object.
-</section>
-
-<section data-dfn-for="WorkerNavigator">
-  <h2>Extensions to the {{WorkerNavigator}} interface</h2>
-
-  <pre class="idl">
+      <h3>
+        <dfn>serial</dfn> attribute
+      </h3>When getting, the {{Navigator/serial}} attribute always returns the
```

Comment:
We probably need to add `<p>` tags around text like this so that the tool doesn't collapse it onto the previous line.

Reply:
I've added `<p>` tags where they seem appropriate